questions/106-medium-trimleft/template.ts

```diff
@@ -1 +1,3 @@
-export type TrimLeft<S extends string> = any
+export type TrimLeft<S extends string> = S extends `${' ' | '\n' | '\t'}${infer R}`
+    ? TrimLeft<R>
+    : S
```

Comment:
I would recommend don't hardcode the union types, you can save them in a type to make the code more readable.
`type WhiteSpace = ' ' | '\n' | '\t'`